src/components/tagsInput/tagsInput.tsx

```diff
@@ -45,21 +56,21 @@ export interface ITagsInputProps {
     // Props with default values
     placeHolder?: string;
     delimiters?: number[];
+    onTagClick?: (tag: ITag) => void;
+    onCtrlTagClick?: (tag: ITag) => void;
+    onShiftTagClick?: (tag: ITag) => void;
+    onCtrlShiftTagClick?: (tag: ITag) => void; 
 }
 
 /**
  * Current state of tags input component
  * @member tags - IReactTag[] - tags used in lower level component
  * @member currentTagColorIndex - rotates initial color to apply to tags
- * @member selectedTag - tag that was most recently double clicked,
```

Comment:
I'd keep selectedTag documentation in here since we are keeping it

Reply:
Good call. I forgot I put that back.

Reply:
Actually I meant to take it out entirely. It won't be used internally in the component. We can discuss more about it later